api/admin.py

```diff
@@ -1,7 +1,7 @@
 """Site administration module."""
 from django.contrib import admin
-from .models import Accomodation, AccomodationPhoto, Food, FoodPhoto, \
-    Lector, LectorPhoto, Payment, Participant, Workshop, WorkshopPhoto, Year
+
+from . import models
```

Comment:
This change is not related to the PR. Please make file them separately next time.